CONTRIBUTING.md

```diff
@@ -1,9 +1,10 @@
 
 ### Expectations
 
-flowMC is developed and maintained in my spare time and, while I try to be responsive, I don't always get to every issue immediately.
-If it has been more than a week or two, feel free to ping me to try to get my attention.
-This is subject to changes as the community grows
+flowMC is developed and maintained in my spare time and, while I try to be
+responsive, I don't always get to every issue immediately. If it has been more
+than a week or two, feel free to ping me to try to get my attention. This is
```

Comment:
By ping me, do you mean quoting you on the issue thread?

Reply:
Done.